assets/scss/_pagination.scss

```diff
@@ -11,20 +11,12 @@
     text-align: center;
     font-weight: 700;
 
-    span {
-      margin: 0;
-      text-align: center;
-      width: 3.2rem;
-    }
+    margin: 0;
+    text-align: center;
+    width: 2.2rem;
```

Comment:
FYI, this width was selected rather arbitrary. This is what the current pagination looks like:

![image](https://github.com/luizdepra/hugo-coder/assets/730342/e897666f-cd62-4bf7-bf4a-4c5f7792dfb0)

Reply:
It looks nice to me. :D